frontend/src/component/project/Project/ProjectOverview.tsx

```diff
@@ -91,10 +88,12 @@ const InfiniteProjectOverview = () => {
                         onChange={refetch}
                         total={total}
                     />
-                    {prevCursors.length > 0 ? (
+                    {currentOffset > 0 ? (
                         <Box onClick={fetchPrevPage}>Prev</Box>
                     ) : null}
-                    {nextCursor && <Box onClick={fetchNextPage}>Next</Box>}
+                    {currentOffset + PAGE_LIMIT < total && (
```

Comment:
Could have more readable name for the variable. Should also use ConditionallyRender until we change the ADR. For now I'd use ternaries when you need to access variables that need typecheck and CR otherwise.